ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParseUri.java

```diff
@@ -268,7 +268,12 @@ private static class InvalidIntegerException extends InvalidUriException {
             .put("out_of_retries", ImmutableSet.of(11)).put("file_error", ImmutableSet.of(20))
             .put("sig_not_trusted", ImmutableSet.of(21, 23))
             .put("sig_unexpected", ImmutableSet.of(22, 23))
-            .put("install_timeout", ImmutableSet.of(30)).build()),
+            .put("install_timeout", ImmutableSet.of(30)).put("unknown_error", ImmutableSet.of(1))
+            .put("sig_check_timeout", ImmutableSet.of(24))
```

Comment:
Optional but it would be slightly more readable if it was one per line, e.g.
```suggestion
            .put("install_timeout", ImmutableSet.of(30)) //
            .put("unknown_error", ImmutableSet.of(1)) //
            .put("sig_check_timeout", ImmutableSet.of(24)) //
```
`//` is for the style checker

Reply:
Thanks, I'll make this change now.

Reply:
Updated!